src/10_functions.py

```diff
@@ -1,12 +1,20 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
-# YOUR CODE HERE
+
+def is_even(num):
+    if num % 2 == 0:
+        return True
+    else:
+        return False
+
+
+print(is_even(2))  # True
+print(is_even(3))  # False
 
 # Read a number from the keyboard
 num = input("Enter a number: ")
 num = int(num)
 
 # Print out "Even!" if the number is even. Otherwise print "Odd"
 
```

Comment:
cleaver. good job

Reply:
You showed a mastery of git workflow by committing every step of the way.
You were able to complete the MVP.
Great work so far. Nothing to complain about.